### ISSUE TITLE
[python-flask] Move `test` directory out of code directory

### ISSUE THREAD

**Issue by alfechner** [OpenAPITools/openapi-generator#14729]
##### Description

In my org we want the structure of Python packages to be consistent. I try to adjust the output of the `python-flask` generator.

After generating I end up with a directory structure like this:

```
/my_service
|_ controllers
|_ models
|_ ...
|_ test
```

I'd like to generate the `test` directory outside of `my_service` and call it `tests`:

```
/my_service
|_ controllers
|_ models
|_ ...
/tests
```

Is that possible?

##### openapi-generator version

`6.0.1`

**Comment by wing328** [comment]
what about move or copy (without override) the directory using command (e.g. mv) post code-generation? that way you will have complete control of the folder location

**Comment by alfechner** [comment]
Good point. I was thinking I'd check if there is a 'native' way to do it. So if somebody can confirm that the requested structure cannot be output by the generator I'd go with pos code-generation logic :)
